lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -23,6 +25,12 @@ def exempt_benefits
       data.fetch(:exempt_benefits)
     end
 
+    def exempted_benefits?
+      exempted_benefits.present? &&
```

Comment:
As discussed, it's fine to leave this for now, because the change is being made in a different Pull Request. We should come back to this at some point and use the commonised code.